composer.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "altis/browser-security",
+    "description": "Browser security utilities for WordPress/Altis",
+    "type": "wordpress-plugin",
+    "require": {
+        "php": ">=7.1",
+    },
+    "license": "GPL-2.0",
+    "authors": [
+        {
+            "name": "Human Made Limited",
+            "email": "hello@humanmade.com"
+        }
+    ],
+    "autoload": {
+        "files": [
+            "namespace.php"
```

Comment:
Presumably this works for `inc/namespace.php` too? Wasn't aware you didn't need to qualify an exact file!

Reply:
Actually already fixed this, just didn't push 😬